Lib/test/test_importlib/test_util.py

```diff
@@ -775,5 +775,33 @@ def test_complete_multi_phase_init_module(self):
             self.run_with_own_gil(script)
 
 
+class MiscTests(unittest.TestCase):
+    def test_atomic_write_should_notice_incomplete_writes(self):
+        from importlib import _bootstrap_external
+        from test.support import os_helper
+        import _pyio
+        import os
```

Comment:
Why the local imports?

Reply:
I've moved them to the global level now, but left `_pyio` local, because it's very unlikely that another test in this file will ever need it.